projects/redemption_configs/configs_specs/configs/specs/config_spec.hpp

```diff
@@ -615,6 +615,8 @@ void config_spec_definition(Writer && W)
         W.member(advanced_in_gui, no_sesman, L, type_<bool>(), "play_video_with_corrupted_bitmap", desc{"Needed to play a video with corrupted Bitmap Update.\nNote: Useless with mpv and mplayer."}, set(false));
 
         W.member(ini_and_gui, no_sesman, L, type_<bool>(), "allow_rt_without_recording", desc { "Allow Realtime display (4eyes) without recording of session" }, set(false));
+
+        W.member(hidden_in_gui, no_sesman, L, type_<int>(), "file_permissions", desc { "Allow to control permissions on recorded files" }, set(440));
```

Comment:
Should be a specific type such as `FilePermission` with personal parser (`redemption_config/redemption_src/configs/{zbuffer,io}.hpp`) and specific additional description (`write_type_info`, `write_*` (track generator errors)).